#goal, empty                
---                             
#result, Odometry array             
nav_msgs/Odometry[] result_odom_array                
---                             
#feedback, empty